i18n/web/en.ts

```diff
@@ -1,12 +1,14 @@
 import { WebTranslations } from '.'
 import { Language, Locale } from '../../utils/api/definitions/enums'
 
+import api_translation from '../api/en'
```

Comment:
is this import used? It doesn't seem to be. if not, please remove. if it is used, please use `camelCase` for all variable names!

Reply:
Yes, it is used for those link URLs. I have changed the variable names